code/modules/nifsoft/software/14_commlink.dm

```diff
@@ -42,7 +42,7 @@
 		..()
 		nif = newloc
 		nifsoft = soft
-		register_device(nif.human)
+		spawn(10) register_device(nif.human) //Need to outwait the other spawn in communicators, sigh.
```

Comment:
Acceptable for now (until `/obj/item/device/communicator/New()` is fixed)  but both that spawn and this spawn should be in initialize() not New()

Reply:
I assume fixing the other New() is contingent upon like. Fixing the order things happen during character spawn which might be "a while", but no I don't like the DOUBLE RACE either.